src/App.tsx

```diff
@@ -15,36 +16,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  None = '',
+  Length = 'length',
+  Alphabetically = 'alphabetically',
+}
+
+const sortList = (
+  goods: string[],
+  sortBy: SortType,
+  isReversed: boolean,
+): string[] => {
+  const sortedGoods = [...goods].sort((a, b) => {
+    switch (sortBy) {
+      case SortType.Alphabetically:
+        return a.localeCompare(b);
+      case SortType.Length:
+        return a.length - b.length;
+      default:
+        return 0;
+    }
+  });
+
+  if (isReversed) {
+    sortedGoods.reverse();
+  }
+
+  return sortedGoods;
+};
```

Comment:
[CODE STYLE] - The function sortList should start with a verb to indicate what it does. Consider renaming it to something like 'getSortedList' to make its purpose clearer.